src/controllers/ratingController.js

```diff
@@ -39,6 +40,31 @@ class ArticleRate {
       data: { rating }
     });
   }
+
+  /**
+     * Check the environment
```

Comment:
What does this mean?

Reply:
It was just for the jsdoc but for real, I found out that it's not relating to the actual function that is being implemented. Let me work on it thanks @mkiterian

Reply:
@mkiterian I've Implemented your feedback